gobblin-core/src/main/java/gobblin/source/extractor/extract/restapi/RestApiExtractor.java

```diff
@@ -73,6 +73,27 @@ public RestApiExtractor(WorkUnitState state) {
 
   protected abstract RestApiConnector getConnector(WorkUnitState state);
 
+  protected void buildQuery(String inputQuery, String entity, JsonArray columnArray) {
```

Comment:
The name is too general, which makes it a function to build any kind of query. Some name like `buildExtractMetadataQuery` is better.  

It's also better to return the query string instead of setting `updatedQuery` inside the function. The value assignment can be done in `extractMetadata`.